v2/inc/c_logging/log_stacktrace.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef LOG_STACKTRACE_H
+#define LOG_STACKTRACE_H
+
+#ifdef __cplusplus
+#include <cstddef>
+#else
+#include <stddef.h>
+#endif
+
+#if defined(_MSC_VER)
+#include "windows.h" /*needed for HANDLE*/
+#else
+#error no other platform supported
+#endif
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+    void getThreadStackAsString(HANDLE thread, char* destination, size_t destinationSize);
```

Comment:
>void getThreadStackAsString(HANDLE thread, char* destination, size_t destinationSize); [](http://example.com/codeflow?start=4&length=86)

Frankly this API has nothing to do with logging. It is merely linked to logging because that's where we put it the first time.

But we actually want to obtain the callstacks in some cases and save the callstack (like we do in EBS when we have a report fault).

I think it's true name is probably get_stacktrace.h/c

/poetry